Modules/II-k6-Foundations/07-Setting-test-criteria-with-thresholds.md

```diff
@@ -1,8 +1,8 @@
-In [Clarifying test criteria](../XX-Future-Ideas/Clarifying-testing-criteria.md#Thresholds), we talked a bit about [thresholds](Performance-Testing-Terminology.md#Threshold) and how they can be used to implement other criteria, such as SLAs.
+In k6, [thresholds](https://k6.io/docs/misc/glossary/#threshold) are used to codify service-level objectives (SLOs); that is, developers and testers using k6 are encouraged to define SLOs for each test as part of the test script options. In this context, thresholds are a way to implement testing criteria rather than testing criteria themselves. Thresholds could also be used to implement service-level Agreements (SLAs).
```

Comment:
```suggestion
k6 encourages developers and testers to define goals for each test. To do this, you can set [thresholds](https://k6.io/docs/using-k6/thresholds) to evaluate whether the test performs to a certain criteria. As an example, you can use thresholds to assert that the system performs within your service-level objectives while the test runs.
```

Reply:
I think we are slowing refining our definition of thresholds, so this is practice. Besides that, they are such an important concept that I think it makes more sense to link to the real doc instead of the glossary entry.

Reply:
Can I add a [tooltip](https://en.wikipedia.org/wiki/Tooltip "This goes to the wikipedia page on tooltips") in Github-flavored markdown?

The preview says I can! From an information-discovery standpoint, I think the most comfortable reader experience would be to use the glossary definition as the tooltip text, and link to the main doc. Not sure if it's worth the effort.